int32 fused_command
string info
string area
string wait_time